src/Draco.Compiler/Internal/Syntax/Syntax.xml

```diff
@@ -401,7 +505,26 @@
     </Field>
   </Node>
 
-  <Node Name="ParameterSyntax" Base="SyntaxNode">
+  <AbstractNode Name="ParameterSyntaxBase" Base="SyntaxNode">
```

Comment:
I personally don't agree with this naming scheme, we don't have elements called "base" in syntax. I would have left `ParameterSyntax` the base type and then have something like `SimpleParameterSyntax` or `NormalParameterSyntax` as derived